### PR TITLE
Add validation for MachinePool/NodePool taints

### PR DESCRIPTION
**Pull request by enxebre**
Relates https://issues.redhat.com/browse/OCPBUGS-10423

### CODE REVIEW

**Review by bryan-cox — APPROVED** [review]
/lgtm

**Comment by andreadecorte** [comment]
lgtm

File: cmd/create/machinepool/machinepool.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package machinepool
 
 import (
 	"fmt"
+	"k8s.io/apimachinery/pkg/util/errors"
```
Comment (vkareh) [inline cmd/create/machinepool/machinepool.go:5]: Can you move these down to the 3rd-party import section below?

**Comment by andreadecorte** [comment]
@enxebre I am extending this in https://github.com/openshift/rosa/pull/1125 since we had other fixes needed wrt taints, this one can be closed

**Comment by gdbranco** [comment]
Closing in favor of PR from @andreadecorte's comment

**Pull request closed without merging**